dataflow/java/src/main/java/com/httparchive/dataflow/BigQueryImport.java

```diff
@@ -125,7 +141,7 @@ public static Response truncateUTF8(String s, int maxBytes) {
                             more += 3 + t.length();
                         }
                 }
-
+                
```

Comment:
Can we strip whitespace? I don't know which IDE you're using, but NetBeans has a nice code formatter that I used when working on this -- I used the defaults, without any customizations.